components/dialog/overview.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Overview
+page_title: Dialog Overview
+description: Overview of the Dialog for Blazor.
+slug: dialog-overview
+tags: telerik,blazor,dialog,overview
+published: True
+position: 0
+---
+
+# Dialog Overview
+
+The Dialog component is a modal popup that displays information to the user. It is fully customizable and extends the `DialogBuilder` to provide you with a way to achieve complex UI popups.
+
+## Basics
+
+To add a Telerik Dialog to your Blazor app:
+
+1. Add the `TelerikDialog` tag.
+1. Provide `Visible` parameter.
+1. Set its `Title`.
+1. Optionally, choose a `ButtonsLayout` (from the `DialogButtonsLayout` enums).
+    * Its default layout is `DialogButtonsLayout.Stretched`. See more in the [Action Buttons article]({%slug  dialog-action-buttons%})).
```

Comment:
These steps should be simple. Move all explanations to the Features section and make "choose a ButtonLayout" a link to it. Is it ButtonsLayout or ButtonLayout - both syntaxes are used.

Reply:
Done.